src/Microsoft.AspNetCore.Identity/SignInManager.cs

```diff
@@ -582,7 +594,7 @@ private ClaimsIdentity CreateIdentity(TwoFactorAuthenticationInfo info)
         }
 
 
-        private async Task<SignInResult> SignInOrTwoFactorAsync(TUser user, bool isPersistent, string loginProvider = null)
+        private async Task<SignInResult> SignInOrTwoFactorAsync(TUser user, bool isPersistent, string loginProvider = null, bool bypassTwoFactor = false)
```

Comment:
The parameter `bypassTwoFactor` is added to the `SignInOrTwoFactorAsync`, however it is not used in the method body. It is a private method so it is not for overriding. Why is this parameter added?